colcon_notification/desktop_notification/notify2.py

```diff
@@ -47,4 +47,5 @@ def notify(self, *, title, message, icon_path=None):  # noqa: D102
         else:
             self._last_notification.update(
                 title, message=message, icon=icon_path)
+        self._last_notification.set_hint('transient', True)
```

Comment:
It should be sufficient to do this in the `if` block, no?

Reply:
I just tested - yes it can be. I wasn't sure at the time if `update` reset these hints or not. It appears not.